### PR TITLE
KAZOO-5039 modb pagination

### PR DESCRIPTION
**Pull request by jamhed**
- applied to cb_notifications as example

### CODE REVIEW

File: applications/crossbar/src/modules/cb_notifications.erl [hunk 1]
```diff
@@ -221,7 +221,7 @@ validate(Context) ->
     validate_notifications(maybe_update_db(Context), ReqVerb).
 
 validate(Context, ?SMTP_LOG) ->
-    load_smtp_log(Context);
+    crossbar_view:load(Context, ?CB_LIST_SMTP_LOG, fun(JObj) -> kz_json:get_value(<<"value">>, JObj) end);
```
Comment (lazedo) [inline applications/crossbar/src/modules/cb_notifications.erl]: `fun(JObj) -> kz_json:get_value(<<"value">>, JObj) end`why not use something like `normalize_view_results ` ?

Comment (jamhed) [inline applications/crossbar/src/modules/cb_notifications.erl]: you mean to name anonymous function? thought it is too simple to have a name.

File: applications/crossbar/src/crossbar_view.erl [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+%%% @contributors
+%%%   Roman Galeev
+%%%-------------------------------------------------------------------
+
+-module(crossbar_view).
+-include("crossbar.hrl").
+-export([load/3, load/5]).
+
+-define(PAGINATION_PAGE_SIZE, kapps_config:get_integer(?CONFIG_CAT, <<"pagination_page_size">>, 50)).
+-define(DEFAULT_RANGE, kapps_config:get_integer(?CONFIG_CAT, <<"maximum_range">>, (?SECONDS_IN_DAY * 31 + ?SECONDS_IN_HOUR))).
```
Comment (icehess) [inline applications/crossbar/src/crossbar_view.erl]: you can use `?MAX_RANGE` which is defined in `crossbar.hrl`

**Comment by icehess** [comment]
@jamhed @lazedo LGTM

**Pull request merged**